core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/source/SourceCollector.java

```diff
@@ -3,41 +3,30 @@
  *
  * Copyright (C) 2013 Cognifide Limited
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
  *
  * http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
  */
 package com.cognifide.aet.job.common.collectors.source;
 
 import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
```

Comment:
The new imports order is good according to the Google styleguide:
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements
however we have it different order in other files...

maybe we could use current convention and discuss what to do with *not so google* order of imports in future?
I mean: it might be better to have it consistent through all of files. What do you think?

Reply:
I guess we should finally have all imports consistent with [formatter we recommend](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions). The problem here is that Google changed formatting approach in one of previous formatters version.
I revert the change in imports order in files of this pull request and let's discuss what to to with it.
Thanks for noticing that :)